wire-ios/Wire-iOS/Sources/UserInterface/Components/Views/LinkInteractionTextView.swift

```diff
@@ -111,11 +111,11 @@ extension LinkInteractionTextView: UITextViewDelegate {
                   shouldInteractWith URL: URL,
                   in characterRange: NSRange,
                   interaction: UITextItemInteraction) -> Bool {
-
         // present system context preview
         if  UIApplication.shared.canOpenURL(URL),
             interaction == .presentActions,
-            !isMarkdownLink(in: characterRange) {
+            !isMarkdownLink(in: characterRange),
+                Settings.isClipboardEnabled {
```

Comment:
indentation looks off

Reply:
Linter did not throw any error

Reply:
it's still visibly off by one tab though

Reply:
I will fix this in `develop` branch